### ISSUE TITLE
store current top5k/top10k as historical "january 2015" data, and rerun?

### ISSUE THREAD

**Issue by Pomax** [redox/top-github-users#4]
This seems like a fun repo to revisit every 6 months to see how the "rankings" have changed, would it be worth putting the current 5k/10k results in a "January 2015" dir, and rerunning the process to see what the most up to date values are?

**Comment by redox** [comment]
Yeah, I actually need to review the underlying BigQuery query because githubarchive.org changed the way they store the data since Jan 1st. Will try to work on it shortly.
